image_provisioner/playbooks/roles/aos-ansible/tasks/main.yaml

```diff
@@ -37,3 +37,18 @@
     name: iptables 
     state: started 
     enabled: yes
+
+- block:
+   - name: check if skopeo is installed
+     command: rpm -q skopeo
+     register: skopeo_status
+
+   - type: "script"
+     title: "copy openshift images from docker storage to CRI-O storage"
+     timeout: 10800
+     script: |-
+          for i in $(docker images registry.ops.openshift.com/openshift3/* --format "{{.ID}}: {{.Repository}}"); do
```

Comment:
this will break soon.  the registry url is changing.  any chance this can somehow be parameterized?

Reply:
Done.